service/controller/v1/prometheus/scrapeconfig.go

```diff
@@ -666,6 +666,12 @@ func getScrapeConfigs(service v1.Service, certificateDirectory string) []config.
 					Regex:        ServiceWhitelistRegexp,
 					Action:       relabel.Keep,
 				},
+				// Drop non-managed kiam pods and keep only Giantswarm managed kiam pods
+				{
+					SourceLabels: model.LabelNames{KubernetesSDPodNameLabel, PodSDGiantswarmServiceTypeLabel},
+					Regex:        KiamPodNameRegexpNonManaged,
+					Action:       relabel.Drop,
+				},
```

Comment:
drop  endpoints of kiam pods that have pod label`giantswarm.io/service-type` empty